app/src/main/java/org/xbmc/kore/service/library/SyncMovies.java

```diff
@@ -163,7 +163,6 @@ public void onSuccess(List<VideoType.DetailsMovie> result) {
                     // As we're going to recurse, these result objects can add up, so
                     // let's help the GC and indicate that we don't need this memory
                     // (hopefully this works)
-                    result = null;
```

Comment:
@poisdeux Yes, there is no need to set it to null because it is out of scope after this method ends and candidate for GC removal.